.github/workflows/snyk.yaml

```
@@ -27,20 +35,23 @@ jobs:
         with:
           version: v3.17.0
 
-      - name: Render Templates for ${{ matrix.chart }}
+      - name: Render Templates for ${{ matrix.chartConfig.name }}
         run: |
           helm repo add bitnami https://charts.bitnami.com/bitnami
           helm repo add backstage https://backstage.github.io/charts
           helm repo update
-          helm dependency build ./charts/${{ matrix.chart }}
-          helm template ./charts/${{ matrix.chart }} --output-dir ./output/${{ matrix.chart }}
 
-      - name: Run Snyk IaC Scan for ${{ matrix.chart }}
+          helm dependency build ./charts/${{ matrix.chartConfig.path }}
+          helm template rhdh ./charts/${{ matrix.chartConfig.path }} \
+            ${{ matrix.chartConfig.cliArgs }} \
+            --output-dir ./output/${{ matrix.chartConfig.name }}
+
+      - name: Run Snyk IaC Scan for ${{ matrix.chartConfig.name }}
         continue-on-error: true
         uses: snyk/actions/iac@b98d498629f1c368650224d6d212bf7dfa89e4bf # 0.4.0
         env:
           SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
           SNYK_ORG_ID: ${{ secrets.SNYK_ORG_ID }}
         with:
-          args: --report --org=$SNYK_ORG_ID --target-name="redhat-developer/rhdh-chart/${{ matrix.chart }}"
-          file: ./output/${{ matrix.chart }}
+          args: --report --org=$SNYK_ORG_ID --target-name="redhat-developer/rhdh-chart/${{ matrix.chartConfig.name }}"
```

```suggestion
          args: --report --target-name="redhat-developer/rhdh-chart/${{ matrix.chartConfig.name }}"
```
@rm3l we already passed SNYK_ORG_ID via the env, and then again in args, the Snyk CLI will already pick up SNYK_ORG_ID from the environment. This is safe but redundant.

---

Is it documented that the Snyk CLI will read the `SNYK_ORG_ID` env var if set? I couldn't find any reference to that. But maybe I missed it.

---

fyi note: [MCP Snyk Server project](https://github.com/sammcj/mcp-snyk), explicitly documents that the Snyk CLI will look for the organization ID in this order:

1. Command argument (--org=...)
2. Environment variable (SNYK_ORG_ID)
3. CLI config (snyk config get org)
https://github.com/sammcj/mcp-snyk?tab=readme-ov-file#organization-id-configuration, just a note we can leave as is :)

---

> fyi note: MCP Snyk Server project, explicitly documents that the Snyk CLI will look for the organization ID in this order:

Yes, but this is specific to this tool, which we are not using here. As depicted in https://docs.snyk.io/snyk-cli/configure-the-snyk-cli/environment-variables-for-snyk-cli, we could indeed use `SNYK_CFG_<KEY>`, but let's leave this for a different issue/PR if you want to do this.